src/test/java/tests/DeleteApplicationDbsTests.java

```diff
@@ -22,16 +24,16 @@
     @Test
     public void testDeleteApplicationDbsView() throws Exception {
         // Create application db by making an install request
-        String appId = "casetestappid";
+        String dbPath = getApplicationDBPath("casetestdomain", "casetestuser", "casetestappid");
```

Comment:
NIT but usually we would do `ApplicationUtils.getApplicationDBPath()` since this is more readable http://stackoverflow.com/a/421127